src/test/java/com/snowflake/kafka/connector/records/RecordContentTest.java

```diff
@@ -271,6 +271,6 @@ public void testColumnNameFormatting() throws JsonProcessingException {
     Map<String, Object> got = service.getProcessedRecordForStreamingIngest(record);
 
     assert got.containsKey("\"NaMe\"");
-    assert got.containsKey("AnSwEr");
+    assert got.containsKey("\"ANSWER\"");
```

Comment:
to understand again, is this because name is double quoted but answer is not?

Reply:
correct